charts/external-secrets/templates/govuk-chat/oauth.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: external-secrets.io/v1beta1
```

Comment:
It doesn't look like other apps need this for signon credentials. I'm not sure how they get them but I have vague memories of there being a system that auto rotates them.

Reply:
that's what i noticed too, I only noticed account api using it. Nice thanks for notifying the team